### DISCUSSION TITLE
Keyboard Shortcut for “Fill, Cover, Fit” Layout Change

### DISCUSSION THREAD

**Discussion by Gales-P** [blinksh/blink#1689] [Ideas]
When using a keyboard and mouse, it would be nice if I didn’t have to rely on the touchscreen to change the “_Fill, Cover, Fit_” layout. Please consider adding a keyboard shortcut to change that. Thank you.

**Comment by yury** [comment]
That is good feature request. I propose `cmd-l`. Any other suggestion?

**Reply by Gales-P** [reply]
That is a perfect default. And add this to the list of actions for custom keyboard shortcuts, so that people can change the shortcut if they need to.
